frontend/src/pages/accounts/settings.page.tsx

```diff
@@ -183,38 +183,49 @@ const Settings: NextPage = () => {
-                          title={l10n.getString("settings-button-copy")}
-                          onClick={copyApiKeyToClipboard}
-                        >
-                          <img
-                            src={copyIcon.src}
-                            alt={l10n.getString("settings-button-copy")}
-                            className={styles["copy-icon"]}
-                          />
-                        </button>
-                        <span
-                          aria-hidden={!justCopiedApiKey}
-                          className={`${styles["copied-confirmation"]} ${
-                            justCopiedApiKey ? styles["is-shown"] : ""
-                          }`}
-                        >
-                          {l10n.getString("setting-api-key-copied")}
+                    <div className={styles["settings-api-key-wrapper"]}>
+                      <input
+                        id="api-key"
+                        ref={apiKeyElementRef}
+                        className={styles["copy-api-key-display"]}
+                        value={profile.api_token}
+                        size={profile.api_token.length}
+                        readOnly={true}
+                      />
+                      <span className={styles["copy-controls"]}>
+                        <span className={styles["copy-button-wrapper"]}>
+                          <button
+                            type="button"
+                            className={styles["copy-button"]}
+                            title={l10n.getString("settings-button-copy")}
+                            onClick={copyApiKeyToClipboard}
+                          >
+                            <img
+                              src={copyIcon.src}
+                              alt={l10n.getString("settings-button-copy")}
+                              className={styles["copy-icon"]}
+                            />
+                          </button>
+                          <span
+                            aria-hidden={!justCopiedApiKey}
+                            className={`${styles["copied-confirmation"]} ${
+                              justCopiedApiKey ? styles["is-shown"] : ""
+                            }`}
+                          >
+                            {l10n.getString("setting-api-key-copied")}
+                          </span>
                         </span>
                       </span>
-                    </span>
+                    </div>
+                    <div className={styles["settings-api-key-copy"]}>
+                      {l10n.getString("settings-api-key-description")}{" "}
+                      <Localized
+                        id="settings-api-key-description-bolded"
+                        elems={{ b: <b /> }}
+                      >
+                        <span className={styles["settings-api-key-copy"]} />
+                      </Localized>
```

Comment:
@Vinnl Did I do this right? It doesn't show bolded for me:

<img width="1181" alt="image" src="https://user-images.githubusercontent.com/13066134/174636747-5adc1aa0-c303-4f83-b13b-7171e623e05e.png">

Reply:
@codemist Yes, that looks right, but the localiser can then use `<b>` in their strings - so you'll have to update `pendingTranslations.ttl` (and the l10n PR) as well.

Reply:
@Vinnl Andrei confirmed that he still isn't seeing the bolded string, is updating `pendingTranslations` necessary on stage?

Reply:
@codemist Hmm, I see this in `app.ftl`:

```
settings-api-key-description = This is your { -brand-name-relay } API Key. It allows other apps to automatically access your { -brand-name-relay } account.
settings-api-key-description-bolded = Only use this with apps that you trust to access your { -brand-name-relay } account for you.
```

I'd have expected something like this?

```
settings-api-key-description-bolded = This is your { -brand-name-relay } API Key. It allows other apps to automatically access your { -brand-name-relay } account. <b>Only use this with apps that you trust to access your { -brand-name-relay } account for you.<b/>
```

(And no, if the string is already in the l10n repository, that should override whatever is in `pendingTranslations.ftl`, so updating the latter shouldn't be necessary.)